sn_node/src/log_markers.rs

```diff
@@ -51,6 +51,12 @@ pub enum Marker<'a> {
     /// Valid spend stored
     ValidSpendPutFromClient(&'a PrettyPrintRecordKey<'a>),
 
+    /// The peer has been considered as bad
+    PeerConsideredAsBad(&'a PeerId),
+
+    /// We have been flagged as a bad node by a peer.
+    FlaggedAsBadNode(&'a NetworkAddress),
```

Comment:
Can't we use `PeerId` instead of these `NetworkAddress`es? Maybe that needs a bit of a refactor in `sn_protocol`'s `cmd.rs`, but would seem neater.

If it's not easy or not possible, fine, lets keep it like this.

Reply:
I think `sn_protocol`'s `cmd.rs` uses `NetworkAddress` everywhere because we don't want struct from other dependency through the wire? 
Anyway, I tried it and `PeerId` does not implement deserialize, so that will not be possible.

Although I can change things to `PeerId` at other places.

Reply:
Sounds complicated, lets leave it out.